EventsExpress/ClientApp/src/components/event/event-item.js

```diff
@@ -93,49 +61,13 @@ export default class EventCard extends Component {
 
                     }}
                 >
-                    <Menu
-                        id="simple-menu"
-                        anchorEl={anchorEl}
-                        keepMounted
-                        anchorOrigin={{
-                            vertical: "bottom",
-                            horisontal: "left"
-                        }}
-                        open={Boolean(anchorEl)}
-                        onClose={this.handleClose}
-                    >
-
-                        {
-                            PrintMenuItems
-                        }
-                    </Menu>
-                    <CardHeader
-                        avatar={
-                            <Button title={owners[0].username} className="btn-custom" onClick={this.handleClick}>
-                                <Badge overlap="circle" badgeContent={owners.length} color="primary">
-                                    <CustomAvatar
-                                        className={classes.avatar}
-                                        photoUrl={owners[0].photoUrl}
-                                        name={owners[0].username}
-                                    />
-                                </Badge>
-                            </Button>
-
-                        }
-
-                        action={
-                            <Tooltip title="Visitors">
-                                <IconButton>
-                                    <Badge badgeContent={countVisitor} color="primary">
-                                        <i className="fa fa-users"></i>
-                                    </Badge>
-                                </IconButton>
-                            </Tooltip>
-                        }
+                    <EventHeader 
+                        members={members}
```

Comment:
Might we redesign behavior, to load all this detailed data separately from the main event info? So It will be a lot easier to load from the db, map, and maintain...
@iuriikhrystiuk, FYI